core/modules/file.py

```diff
@@ -512,30 +512,41 @@ def initializeUpload(self,
         return db.encodeKey(fileSkel["key"]), uploadUrl
 
     @exposed
-    def getUploadURL(self, fileName, mimeType, size=None, skey=None, *args, **kwargs):
+    def getUploadURL(self, fileName: str, mimeType: str, size: int = None, skey: str = None, *args, **kwargs):
         node = kwargs.get("node")
         authData = kwargs.get("authData")
         authSig = kwargs.get("authSig")
-        # Validate the the contentType from the client seems legit
-        mimeType = mimeType.lower()
-        assert len(mimeType.split("/")) == 2, "Invalid Mime-Type"
-        assert all([x in string.ascii_letters + string.digits + "/-.+" for x in mimeType]), "Invalid Mime-Type"
+
+        # Validate the contentType from the client seems legit
+        mimeType = mimeType.strip().lower()
+        if not (
+            mimeType
+            and mimeType.count("/") == 1
+            and all([ch in string.ascii_letters + string.digits + "/-.+" for ch in mimeType])
```

Comment:
```suggestion
            and all(ch in string.ascii_letters + string.digits + "/-.+" for ch in mimeType)
```

Reply:
Didn't know this is valid, but it seems to work...